src/circleci/rollcage/core.clj

```diff
@@ -176,20 +177,26 @@
   ([access-token options]
    (client* access-token options)))
 
+(def ^:private rollbar-to-logging
+  "A look-up table to map from Rollbar severity levels to tools.logging levels"
+  {"critical" :fatal
+   "error"    :error
+   "warning"  :warn
+   "info"     :info})
+
 (defn notify
   ([level client exception]
    (notify level client exception {}))
   ([level client exception {:keys [url params]}]
-   (if (string/blank? (:access-token client))
-     ;; Logging exception where rollcage client is not configured properly e.g.
-     ;; development environments
-     (logging/warn exception "Rollbar token is empty, falling back to logging.")
-     (let [params (merge params
-                         (when (instance? clojure.lang.ExceptionInfo exception)
-                          {:ex-data (ex-data exception)}))]
-       (send-item endpoint
-                  (make-rollbar client level exception url params)
-                  (:result-fn client))))))
+   (let [log-level (rollbar-to-logging level)
+         params (merge params (ex-data exception))]
+     (if (string/blank? (:access-token client))
+       (logging/logf log-level exception "No Rollbar token configured. Not reporting exception: %s" exception)
+       (do
+         (logging/logf log-level exception "Sending exception to Rollbar: %s" exception)
```

Comment:
~ Does the exception need to be added to the log string given that it's also passed as an argument so will get logged?